python/pdstools/app/health_check/Home.py

```diff
@@ -1,9 +1,9 @@
 import shutil
 
-import requests
 import streamlit as st
 
 import pdstools
+from pdstools.utils.streamlit_utils import get_latest_pdstools_version
```

Comment:
Last remark: I don't think this should be in streamlit_utils, it's not streamlit specific

Reply:
Ah I see, you want it to be cached. Maybe it could be better to use some other caching mechanism so it's streamlit agnostic?

Reply:
Right, I used streamlit's caching decorator, will try to use another way of caching

Reply:
Carried the original function to cdh_utils and calling this from streamlit_utils now.
```
@st.cache_data
def st_get_latest_pdstools_version():
    return cdh_utils.get_latest_pdstools_version()

```